source/tomopy/recon/algorithm.py

```diff
@@ -392,10 +415,29 @@ def _dist_recon(tomo, center, recon, algorithm, args, kwargs, ncore, nchunk):
             # run in this thread (useful for debugging)
             algorithm(tomo[slc], center[slc], recon[slc], *args, **kwargs)
     else:
-        # execute recon on ncore threads
-        with cf.ThreadPoolExecutor(ncore) as e:
-            for slc in use_slcs:
-                e.submit(algorithm, tomo[slc], center[slc], recon[slc], *args, **kwargs)
+        # execute recon on ncore processes. Accelerated methods have internal thread-pool
+        # and NVIDIA NPP library does not support simulatenously leveraging multiple devices
+        # within the same process
+        if "accelerated" in kwargs and (kwargs["accelerated"] is True
+                                        or kwargs["accelerated"]):
```

Comment:
```suggestion
        if "accelerated" in kwargs and kwargs["accelerated"]:
```
With Python there is a subtle difference between identity (`is`) and equality (`==`). For example:

```
True is True
not (True is np.array(True))
True == np.array(True)
True == 1
not (True is 1)
```
The existence of `array(True)` means that somewhere in the stack, the inputs are being converted to NumPy arrays.